react/modules/gaEvents.ts

```diff
@@ -118,3 +119,34 @@ export function selectPromotion(eventData: PixelMessage['data']) {
 
   updateEcommerce(eventName, data)
 }
+
+export function addToCart(eventData: PixelMessage['data']) {
+  if (!shouldMergeUAEvents()) return
+
+  const eventName = 'add_to_cart'
+
+  const {
+    currency,
+    items: [item],
+  } = eventData
+
+  const productName = getProductNameWithoutVariant(item.name, item.skuName)
+  const formattedPrice =
+    item.priceIsInt === true ? item.price / 100 : item.price
+
+  const data = {
+    currency,
+    value: formattedPrice,
+    items: {
```

Comment:
`items` must be a array according to [docs](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?hl=en&client_type=gtag#add_to_cart_item)